src/desktop/org.opensuse.yast.Migration-SLE.desktop

```diff
@@ -0,0 +1,22 @@
+[Desktop Entry]
+Type=Application
+Categories=Settings;System;Qt;X-SuSE-YaST;X-SuSE-YaST-Software;
+
+X-KDE-ModuleType=Library
+X-SuSE-YaST-Call=migration-sle
+
+X-SuSE-YaST-Group=Software
+X-SuSE-YaST-Argument=
+X-SuSE-YaST-RootOnly=true
+X-SuSE-YaST-Geometry=
+X-SuSE-YaST-SortKey=
+X-SuSE-YaST-Keywords=sle,sles,enterprise,migration,registration
+
+Icon=yast-migration
+Exec=xdg-su -c "/sbin/yast2 migration-sle"
+
+Name=YaST Online Migration to SLES
+
+GenericName=Online SLES Migration
+Comment=Migrate to the SUSE Linux Enterprise Server product
```

Comment:
Question, only to SLES or to SLE in general? I mean, SLE Desktop and other products.

Reply:
Ok, answered in https://github.com/yast/yast-migration-sle/pull/1/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R14

> This YaST module allows online migration openSUSE Leap distribution to the
SUSE Linux Enterprise Server (SLES).

Reply:
Currently only SLES, but in theory any SLE product, depends on the SCC.